www/artists/jmitchell2/style2.css

```diff
@@ -17,10 +17,13 @@ h1{
 	color: yellow;
 }
 
-a:link{
-	color: yellow;
+.title{
+	text-align: center;
+	color: white;
 }
 
-a:hover{
-	color: pink; 
-}
\ No newline at end of file
+a, a:visited{
```

Comment:
As a general rule, we prefer to preserve the original coloration of the sites. Would it be possible to make the links yellow normally, and pick on hover, as it originally was? 
Also, colouring the links white with no class restriction is probably a bad idea - there are links on index.html, which has a white background. Are the links even visible there with this change?

Reply:
there are 2 style sheets and the one i edited was for the artwork and paintings only, i can revert it back to being yellow and pink for those two. I decided to only modify those 2 pages as they are the only ones where text was hard to read and preserving the other pages is important